CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -1,5 +1,21 @@
 # Changelog 2.3
 
+## 2.3.1 - 2025-07-03
+
+### Changed
+
+- Update go-ipfix to v0.14.0. ([#7080](https://github.com/antrea-io/antrea/pull/7080), [@antoninbas])
+- Document SecondaryNetwork support for SR-IOV. ([#7076](https://github.com/antrea-io/antrea/pull/7076), [@tnqn])
+
+### Fixed
+
+- Enhance OVS commands for Antrea Windows to accelerate container recovery after OVS processes restart and improve robustness. ([#7228](https://github.com/antrea-io/antrea/pull/7228), [@XinShuYang])
+- Sync groups members for Pods that turns into terminated status to ensure released IPs are excluded in time for Antrea NetworkPolicy. ([#7217](https://github.com/antrea-io/antrea/pull/7217), [@Dyanngg])
```

Comment:
it's not just about IP addresses actually, since AppliedToGroups should be correct too

I'd suggest:

> Sync affected groups in the Antrea Controller when a Pod goes into Terminated state, to ensure that the required updated are sent immediately to Agents.